sematic/api/endpoints/runs.py

```diff
@@ -431,17 +431,24 @@ def save_graph_endpoint(user: Optional[User]):
     graph = flask.request.json["graph"]
```

Comment:
Is this the right moment to add user info? 

It seems the runs were created earlier than `PUT /api/v1/graph`. If a `GET /api/v1/run` was requested between run creation and `PUT /api/v1/graph` call, it might not have the `user` info. 

Or is this the first moment a run would get created?

Reply:
This is where runs get created the very first time. There is no other run creation endpoint.

Reply:
`/api/v1/resolutions/<resolution_id>/rerun` clones the root run, creating a new one. We should either move the user-generating info somewhere more specific, or copy this info there.

Reply:
Done.